source/unified-test-format/tests/valid-pass/entity-find-cursor.yml

```diff
@@ -84,5 +84,5 @@ tests:
               commandName: killCursors
           - commandSucceededEvent:
               reply:
-                cursorsKilled: { $$type: array }
```

Comment:
It's quite possible Go simply wasn't running this test on a pre-3.2 server. An alternative solution may be to specify a runOnRequirement such that this only runs when `killCursors` is actually used. 

If this line stays, please add a comment above this line explaining why you're using `$$unsetOrMatches`. Also, please correct the capitalization: [`$unsetOrMatches`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#unsetormatches). The spec notes when certain things are case-insensitive (e.g. `errorContains`), but we should assume case matters otherwise.

Reply:
Done